### ISSUE TITLE
Missing Pools from Query

### ISSUE THREAD

**Issue by nicholaswood** [sushiswap/sushiswap-subgraph#91]
This problem seems to have started a few days ago, the deployed subgraph seems to be missing pools on the MATIC network.  For example, trying to query for the USDC-WETH pool (one of the most active) produces no results:

On sushi analytics: https://app.sushi.com/analytics/pools/0x34965ba0ac2451a34a0471f04cca3f990b8dea27?chainId=137
On polygonscan: https://polygonscan.com/token/0x34965ba0ac2451a34a0471f04cca3f990b8dea27
On sushiswap subgraph: https://thegraph.com/hosted-service/subgraph/sushiswap/matic-exchange
query: 
{
        pair(id:"0x34965ba0ac2451a34a0471f04cca3f990b8dea27") 
        {
            id
        }
    }
 
result:
{
  "data": {
    "pair": null
  }
}

I did notice if I switch to the "pending version" of the subgraph the I get a result to the query - was this a recently fixed issue?

**Comment by jiro-ono** [comment]
Yeah sorry about this, there is actually a bug from a recent deployment into the prod version of the subgraph. Working with the graph team currently on restore that old one, and getting it back into this endpoint. Will probably be a few days or longer before it's gets all fixed.

**Comment by LiquidICP** [comment]
is this a problem in mainnet too?
I just checked for usdt on mainnet it results in null as well
